src/components/forms/NetworkDevicesForm/NetworkDeviceContent.tsx

```diff
@@ -82,6 +89,13 @@ const NetworkDeviceContent: FC<Props> = ({
         formik={formik}
         canSelectManualAcls={supportsNicDeviceAcls(network)}
       />
+      <NetworkDeviceIPAddress
+        formik={formik}
```

Comment:
I suspect this will leak it nic device configuration for profiles as well. We should add a condition to only have it show for instances.